### ISSUE TITLE
Comboniani Collection

### ISSUE THREAD

**Issue by smaugustine** [BetaMasaheft/Documentation#1478]
I noticed that there are no manuscripts from the Vatican Comboniani Collection and Raineri's catalogue is not listed in the list of manuscript catalogues. Is this just that no manuscripts from the collection have been added yet, or is there a reason the collection is excluded?

I previously did some work on [Comb.O.24](https://digi.vatlib.it/view/MSS_Comb.O.24), which is fully digitized in colour, and could add the data from Raineri's catalogue (the MS is a copy of ልሳነ፡ ሰብእ፡/ChRep314).

**Comment by PietroLiuzzo** [comment]
My guess is the second hypothesis. so, great if you can add those now or later, thanks!

**Comment by PietroLiuzzo** [comment]
the comb is available via IIIF, so, please add that linking as well in the new file. Hopefully the section of the guidelines about this is still correct https://betamasaheft.eu/Guidelines/?q=manifest&id=images we will see.

**Comment by PietroLiuzzo** [comment]
yep, looks like it is, the manifest link is this one https://digi.vatlib.it/iiif/MSS_Comb.O.24/manifest.json

**Comment by smaugustine** [comment]
What format should the IDs have? `BAVcomb` + letter + number (e.g. `BAVcombo24`, `BAVcombf22`)? Should they be in a new `comb` folder in `VaticanBAV`?

**Comment by PietroLiuzzo** [comment]
https://betamasaheft.eu/Guidelines/?id=entities-id-structure

**Comment by PietroLiuzzo** [comment]
yep, new folder
